tests/kafkatest/services/security/security_config.py

```diff
@@ -120,13 +123,15 @@ def setup_node(self, node):
 
         if self.has_sasl:
             node.account.ssh("mkdir -p %s" % SecurityConfig.CONFIG_DIR, allow_fail=False)
-            jaas_conf_file = self.sasl_mechanism.lower() + "_jaas.conf"
+            jaas_conf_file = "jaas.conf"
```

Comment:
@ewencp Thank you for the review. Since we now support multiple SASL mechanisms in a broker, we need to have a jaas config file that includes all the supported mechanisms. Hence I modified jaas.conf to include sections based on the enabled mechanisms, rather than have one config file per mechanism. I removed the mechanism name from the file since it is no longer associated with a single mechanism.